README.md

```diff
@@ -510,6 +511,43 @@ data:
 You're welcome to raise issues related to feature requests, bugs, or give us general feedback on this project's GitHub Issues page. 
 The SAP BTP service operator project maintainers will respond to the best of their abilities. 
 
+[Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
+
+## Deleting the Operator
+
+To delete the operator, run the following command:
+`helm uninstall <release name> -n <name space>`
+
+Example: 
+
+ >   ```
+  >   helm uninstall sap-btp-operator -n sap-btp-operator
+
+
+**Note:** All service bindings and service instances are also deleted during this process.
+
+#### Possible Issues
+
+   - The deletion of instances and bindings takes more than 5 minutes
+   
+     There are two scenarios and their respective resolutions:
+     
+      - If you get a timeout error, manually re-trigger the delete process.
+      
+      - If you get the following error: `Warning: Hook pre-delete failed: jobs.batch "pre-delete-job" already exists.` please wait for the job to finish.
```

Comment:
what is the indication the the job finished? how can the user check it?

Reply:
Or looking in the cluster jobs and see no pre-delete exist
Or re-trigger the delete command and it doesn't return error "the script is already running"